Spring.FluentContext/FluentXmlApplicationContext.cs

```diff
@@ -0,0 +1,58 @@
+using Spring.Context;
+using Spring.Context.Support;
+using Spring.Core.IO;
+using Spring.Objects.Factory.Config;
+using Spring.Objects.Factory.Support;
+
+namespace Spring.FluentContext
+{
+    public class FluentXmlApplicationContext : XmlApplicationContext, IFluentConfigurableApplicationContext
```

Comment:
I am curious to see the use case for this class. 

Why do you want to mix fluent with xml ?
* do you want to have some config in xml and then other added fluently? I guess this can be done via composing contexts, i.e. `new FluentApplicationContext(parentXmlContext)`,
* do you want to register it in the app.config and then configure at runtime? What is the problem then of just constructing `FluentApplicationContext` at runtime and registering it ?

I still cannot grasp what does it solve...

Reply:
Can make easier switch from XML configs to Fluent as you don't need to register them all at once. Eg. I haven't found a way how to create a `string` object fluently. But probably this class was a semi-product during creating Web implementation.